src/packages/migrate/src/__tests__/__helpers__/captureStdout.ts

```diff
@@ -0,0 +1,34 @@
+class CaptureStdout {
```

Comment:
Note: We also have this https://github.com/prisma/prisma/blob/master/src/packages/cli/src/__tests__/__helpers__/context.ts